### DISCUSSION TITLE
我尝试在Windows10 的lts上绘制TextEditor不兼容

### DISCUSSION THREAD

**Discussion by coder163** [slint-ui/slint#6998] [Q&A]
### Bug Description

我就是按照官方文档的代码来的，版本是slint1.8，发现无法绘制，我不知道具体原因

### Environment Details

- Slint Version:
- Platform/OS:
- Programming Language:
- Backend/Renderer:

**Comment by ogoffart** [comment]
I translated using machine translation:

> ### I'm trying to draw on Windows 10 lts TextEditor is not compatible
> I just followed the code from the official documentation, version slint1.8, and found that I can't draw, I don't know the exact reason why

But I don't really understand that.
We would need more context.

What exactly are you trying to do? What part of the documentation exactly did you follow. What is not working?

**Comment by coder163** [comment]
非常感谢你能回复，我使用的也是机器翻译，代码如下
`import { TextEdit } from "std-widgets.slint";
export component Example inherits Window {
    width: 200px;
    height: 200px;
    TextEdit {
        font-size: 14px;
        width: parent.width;
        height: parent.height;
        text: "Lorem ipsum dolor sit amet,\n consectetur adipisici elit";
    }
}`
系统环境已经没了，我把系统换掉了。简单的说就是没有出现编辑框

**Comment by coder163** [comment]
其他环境我可以保证是正确的，因为控件和事件都有响应

**Comment by coder163** [comment]
同时我在Mac和其他的Windows10非lts系统上测试是正常的，只有lts，所以我想知道slint是不是针对ls有什么限制

**Comment by ogoffart** [comment]
The problem you're describing sounds a bit like https://github.com/slint-ui/slint/issues/5089
We do not have a solution to this problem. It might be related to a bug in `winit`, the library we're using to show the window.
